packages/datagateway-download/src/downloadStatus/adminDownloadStatusTable.component.test.tsx

```diff
@@ -70,12 +70,17 @@ describe('Admin Download Status Table', () => {
     jest.clearAllMocks();
   });
 
-  it('should render correctly', () => {
+  it('should render correctly', async () => {
     const mockedDate = new Date(Date.UTC(2020, 1, 1, 0, 0, 0)).toUTCString();
     global.Date.prototype.toLocaleString = jest.fn(() => mockedDate);
 
     const { asFragment } = renderComponent();
 
+    // wait for data to finish loading
+    expect(
+      await screen.findByText(mockedDate.toLocaleString())
+    ).toBeInTheDocument();
+
```

Comment:
Hi Kenneth, as I'm sure you've seen, this results in a big addition to the snapshot for this test? Am I right in saying this is because the old snapshot was totally wrong and just described an empty table with no rows??? 🤦

Reply:
Yes, that is correct. The old snapshot doesn't wait for the table to finish loading all the data so it is essentially just a snapshot of its loading state, which is not very useful.

Reply:
Oh dear 🤦 thank you for spotting this. It is so important to religiously check snapshots before changing them because they're our best guess of exactly what the DOM looks like. Plus, it's not like computers can check them!

Reply:
I think in general [snapshot testing is not recommended](https://kentcdodds.com/blog/effective-snapshot-testing) for any snapshot bigger than a couple lines, but I kept it because it is already there.